components/image/slider/src/index.js

```diff
@@ -0,0 +1,60 @@
+import React, {PropTypes} from 'react'
+import ReactSlidy from 'react-slidy'
+import SuiMultimedia from '@schibstedspain/sui-multimedia'
```

Comment:
Then we should discuss it more deeply in order to try to use only SUIStudio-based components.